src/App.js

```diff
@@ -1,25 +1,24 @@
-import logo from './logo.svg';
-import './App.css';
+import React, { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
 
-function App() {
+import { Route, BrowserRouter as Router, Routes } from 'react-router-dom';
+import { fetchUserData } from './redux/user/userSlice';
+import Great from './components/great';
+
+const App = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchUserData());
+  }, [dispatch]);
```

Comment:
- [ ] I think it would be great if you just keep this functionality inside of your great component since this is for connecting with redux to display the messages but the app is functionality to just have the component inside it so kindly move them to the great component 🔥🙌✅

Reply:
Thank you for the review, but every time I move it there, the API displays nothing, but when I put it there the API work